pkg/app/server/option.go

```diff
@@ -270,6 +270,13 @@ func WithTraceLevel(level stats.Level) config.Option {
 	}}
 }
 
+// WithAutoRender sets the render type
+func WithAutoRender(b bool) config.Option {
```

Comment:
Why is this named `WithAutoRender` when what is actually does is `AutoReload` templates

Reply:
i don't find proper name for it。maybe autoReload?

use `html template` and when you change the `html` file you have to restart the server. use `AutoRender` you can just make a new request and `html` template will be reload.

i will show the example later